ui/src/app/pages/workspace/workspace-nav-bar.spec.tsx

```diff
@@ -47,13 +47,79 @@ describe('WorkspaceNavBar', () => {
   it('should navigate on tab click', () => {
     const wrapper = component();
 
+    wrapper.find({'data-test-id': 'Analysis'}).first().simulate('click');
+    expect(mockNavigate).toHaveBeenCalledWith(
+      ['workspaces', workspaceDataStub.namespace, workspaceDataStub.id, 'notebooks']);
+
     wrapper.find({'data-test-id': 'Data'}).first().simulate('click');
     expect(mockNavigate).toHaveBeenCalledWith(
       ['workspaces', workspaceDataStub.namespace, workspaceDataStub.id, 'data']);
+
+    wrapper.find({'data-test-id': 'About'}).first().simulate('click');
+    expect(mockNavigate).toHaveBeenCalledWith(
+      ['workspaces', workspaceDataStub.namespace, workspaceDataStub.id, 'about']);
+  });
+
+  const setNeedsReviewPrompt = (needsReviewPrompt: boolean) => {
```

Comment:
mob: we were setting these values incorrectly.

Adding `!!` to various locations in restrictTab() was not a real fix, so I have reverted that change.